libs/libdaemon/src/BaseDaemon.cpp

```
@@ -775,14 +781,22 @@ void BaseDaemon::initialize(Application& self)
 		  * Делаем это после buildLoggers, чтобы не менять текущую директорию раньше.
 		  * Это важно, если конфиги расположены в текущей директории.
 		  */
-		Poco::File opt_cores = "/opt/cores";
-
-		std::string core_path = config().getString("core_path",
-			opt_cores.exists() && opt_cores.isDirectory()
-				? "/opt/cores/"
-				: (!log_path.empty()
-					? log_path
-					: "/opt/"));
+
+		std::string core_path = config().getString("core_path", "");
+		if (core_path.empty())
+			core_path = getDefaultCorePath();
+
+		try {
```

No need to catch. Let it fail.